recovery/redis_users_recovery.py

```diff
@@ -82,11 +82,11 @@ def __init__(self, script_conf: BaseScriptConfig = ScriptConfig(), config: Confi
         self.start_time = None
         self.args = script_conf.args
         self.config = config
```

Comment:
`config` is not used anywhere outside this `__init__` method so it don't need to be in `self`